crates/eframe/src/native/wgpu_integration.rs

```diff
@@ -702,11 +702,6 @@ impl<'app> WgpuWinitRunning<'app> {
             viewport_from_window,
         );
 
-        // Prune dead viewports:
-        viewports.retain(|id, _| active_viewports_ids.contains(id));
-        viewport_from_window.retain(|_, id| active_viewports_ids.contains(id));
-        painter.gc_viewports(&active_viewports_ids);
```

Comment:
What happened with this code?

Reply:
> What happened with this code?

It is not necessary as it is handled in `handle_viewport_output()`.